docs/understand/object-model.md

```diff
@@ -139,6 +139,10 @@ below, "A", "B" and "C" are possible file contents, "X" is a missing file, and "
 | A           | A             | X                  | X          | File deleted on one side                       |
 | A           | X             | A                  | X          | File deleted on one side                       |
 
+The API and lakectl enables passing optional `strategy` flag with the following values: 
+- dest-wins - in case of a conflict, merge will pick the destination
+- source-wins - in case of a conflict, merge will pick the source
+
```

Comment:
I would also add that it's applicable to all conflicting files. i.e. the user can't choose the strategy per file.